third_party/github.com/browserify/browserify/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_binary")
+
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
```

Comment:
include a comment saying where the files came from (including the tag/sha)

Reply:
👍 :